config.yaml

```diff
@@ -0,0 +1,27 @@
+document: modularity-testing
```

Comment:
What is the purpose of this file? Is it referenced when running the avocado test scripts from the command line? If so, can it be used to set the default module type and docker image name so they don't have to manually set in the environment (eg., URL="docker=base-runtime-smoke" MODULE=docker ...)?